### ISSUE TITLE
[Bug]: "60 seconds Installation" cannot deploy the project correctly.

### ISSUE THREAD

**Issue by Plus-xu** [StanGirard/quivr#1816]
### What happened?

OS: Debian 5.10.197-1 (2023-09-29) x86_64
[docker-compose.yml](https://github.com/StanGirard/quivr/blob/b8f8851ccd0a2f058d23b6c8b029d958a3683f15/docker-compose.yml)
d0d74cecbb69bed7315533741c6185f699236bfb
ef53590b58fe3902eb6abc311ed8de6818b757d0

"60 seconds Installation" cannot deploy the project correctly. I used an initialized VPS with Debian 11 installed for deployment. I followed the steps of "60 seconds Installation" for deployment completely, only updated OPENAI_API_KEY in the .env file. After completing the installation, when I log in to http://localhost:8000/project/default/auth/users and try to create a new user, I encounter the error "**Failed to create user: An error has occurred: Failed to fetch.**" I am unable to create any users, and there are no users present in the Users as well. I am also unable to log in using the default login credentials "admin@quivr.app" and "password "admin" at http://localhost:3000/login.

I have attempted multiple times with both the latest version and yesterday's version, and I am encountering the same error.

![image](https://github.com/StanGirard/quivr/assets/52371609/fdbdc7a1-1127-4482-b2ec-f7878d058711)

### Relevant log output

```
supabase-kong | 1.6.4.6 - - [05/Dec/2023:02:37:46 +0000] "GET /api/projects/default HTTP/1.1" 304 0 "http://2.2.2.2:8000/project/default/auth/users" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36"
supabase-kong | 1.6.4.6 - - [05/Dec/2023:02:37:46 +0000] "GET /api/props/project/default/api HTTP/1.1" 304 0 "http://2.2.2.2:8000/project/default/auth/users" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36"
supabase-kong | 1.6.4.6 - - [05/Dec/2023:02:37:46 +0000] "GET /api/projects HTTP/1.1" 304 0 "http://2.2.2.2:8000/project/default/auth/users" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36"
supabase-kong | 1.6.4.6 - - [05/Dec/2023:02:37:46 +0000] "GET /api/integrations/default-org-slug?expand=true HTTP/1.1" 404 2510 "http://2.2.2.2:8000/project/default/auth/users" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36"
supabase-kong | 1.6.4.6 - - [05/Dec/2023:02:37:46 +0000] "GET /api/auth/default/users?limit=10&offset=0&keywords= HTTP/1.1" 304 0 "http://2.2.2.2:8000/project/default/auth/users" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36"
```

**Comment by Plus-xu** [comment]
> @Plus-xu did @dosubot solve the issue ?

NEXT_PUBLIC_BACKEND_URL and SUPABASE_URL  are correctly set to the IP address.
After watching the installation video and reviewing the '60 Seconds Installation' guide, I noticed that the Docker Compose installation method does not include steps for modifying the SUPABASE_SERVICE_KEY, NEXT_PUBLIC_SUPABASE_ANON_KEY, POSTGRES_PASSWORD, JWT_SECRET, and DASHBOARD_USERNAME settings. Should I be adjusting these settings myself? If so, where might I find a tutorial for this? I have consulted the official help documentation but couldn't find relevant instructions. Thank you for your assistance..

**Comment by thams** [comment]
Observing the same behavior, running on Ubuntu at both 27b2df88 and current HEAD 7aec346

**Comment by thams** [comment]
Looking at the Browser console I see the following. Something is assuming the server is running on the same IP as the browser?

```
localhost:8000/auth/v1/admin/users:1 
Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
```

**Comment by thams** [comment]
Ah, configuration required when not running on localhost?

 API_EXTERNAL_URL=http://localhost:8000

**Comment by thams** [comment]
changing all the places in .env that referred to localhost solved it for me.

**Comment by thams** [comment]
Unknown if this will lead to another problem, but when accessing the supabase instance the web browser also is reporting this error.
`http://<ipaddress>:8000/api/integrations/default-org-slug?expand=true`

**Comment by gobegi** [comment]
I was getting the issue that @thams was refering to for a couple of days until I went to jwt.io and verified the JWT tokens that I generated uisng https://supabase.com/docs/guides/self-hosting/docker.  It turns out that if I changed the secret key, the generate JWT token did NOT create a valid JWT token! I pasted the token generated into jwt.io and pasted the secret key that I wanted to use into it and it re-generated the ANON_KEY and then did the same with the SERVICE_KEY.  I no longer get an JWT not verified error when creating a new user ... I now hav successfully created a new user and I'm sure cleared up other functions as well.  :-)
